packages/imperative/__tests__/config/cmd/init/init.handler.test.ts

```diff
@@ -683,7 +992,6 @@ describe("Configuration Initialization command handler", () => {
         // initForDryRun
         const initForDryRunSpy = jest.spyOn(handler as any, "initForDryRun");
 
-        jest.spyOn(process, "cwd").mockReturnValueOnce(__dirname);
```

Comment:
Again, curious about this change? 🙂

Reply:
Again, I removed it unintentionally. I added it back.